src/components/order-service/order-service.controller.ts

```diff
@@ -94,7 +96,44 @@ export class OrderServiceController implements angular.IController {
     this.ctrl.selectedPlan = plan;
   }
 
+  public invalidNewProject() {
+    // test whether selectedProject is actually a newProject obj. and if passed name validation
+    return this.isNewProject() && !this.ctrl.selectedProject.validName;
+  }
+
   public provisionService() {
+    if (this.isNewProject()) {
+      // the selectedProject is actually a newProject object
+      let newProjName = this.ctrl.selectedProject.metadata.name;
+      let newProjDisplayName = this.ctrl.selectedProject.metadata.annotations['new-display-name'];
```

Comment:
In the case where a user is creating a new project, the 'selected project' is the createNewProjObj used in the dropdown which has it's `openshift.io/display-name` set to 'Create Project'.  'new-display-name' contains user input for the new proj's displayName.